modules/ept-frontend/client/components/post_processing/post-processing.directive.js

```diff
@@ -89,11 +89,7 @@ module.exports = ['$timeout', '$filter', '$compile', function($timeout, $filter,
         var classString = styleString.replace('class="bbcode-', '');
         classString = classString.replace('"', '');
 
-        if (classString.indexOf('color-_') === 0) {
```

Comment:
@akinsey I've re-added the `color-_` if statement (and necessary logic) as I just noticed what it actually does. Must have been tired when I removed it. 

Also fixed how glow and shadow is processed as it didn't allow basic names like `red` after my last few commits. I'm still very much in favor of people entering whatever they want. If it doesn't work they'll see it soon enough.